libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -929,8 +937,10 @@ bool GCS_MAVLINK::should_send_message_in_delay_callback(const ap_message id) con
     // microseconds to return!
 
     switch (id) {
-    case MSG_HEARTBEAT:
     case MSG_NEXT_PARAM:
+        return is_high_latency_link ? false : true;
```

Comment:
I'm not sure why we need code here for `MSG_NEXT_PARAM`.

This method is more about whether it is OK to send a message while we're delaying, NOT whether we should send a message at all.  i.e. this should return true but there should never be anything to send.

Reply:
Removed.